site/content/blog/2020-11-01-whats-new-in-svelte-november-2020.md

```diff
@@ -0,0 +1,43 @@
+---
+title: What's new in Svelte: November 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+BLURB SUMMARIZING WHAT'S NEW...
+"What's new in Svelte" series! We'll try to make this a monthly blog post in which you'll find out about new features, bug fixes, and a showcase of Svelte projects from around the community.
+
+## New features
+1. 
```

Comment:
Slot forwarding might be a notable new feature, but it seems broken for some cases, we might need to see if #5536 lands before november.